### PR TITLE
home page edited

### PR DESCRIPTION
**Pull request by prihit**

### CODE REVIEW

File: base/views.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def custom_internal_error(request, *args, **argv):
 def home(request):
     colleges = College.objects.filter(
         is_top=True,
-    )[:9]
+    )
+    print(colleges)
```
Comment (chayan007) [inline base/views.py:22]: remove this !

**Comment by prihit** [comment]
Annotation 2021-01-30 124339

File: templates/layouts/public.html [hunk 2]
```diff
@@ -39,6 +39,8 @@
           <div class="jumbotron">
             <div class=" container p-l-0 p-r-0   container-fixed-lg sm-p-l-0 sm-p-r-0">
               <div class="inner">
+                <a href="#" class="search-link d-lg-inline-block d-none" data-toggle="search"><i class="pg-icon">search</i>Type
```
Comment (chayan007) [inline templates/layouts/public.html:42]: revert the search, will cover in different PR

`git checkout origin/main -- templates/layouts/public.html`

File: base/views.py (hunk 1 above)
Comment (prihit) [reply base/views.py:22]: done

File: templates/pages/public/home.html [hunk 3]
```diff
@@ -79,1124 +79,52 @@ <h3 class="text-black">Psychometric test</h3>
...
-            <!-- START card -->
-            <div class="card card-transparent">
-              
               <div class="m-0 row card-body">
-                <div class="col-lg-4 no-padding">
+                <div class="col-lg-6 no-padding">
                   <div class="p-r-30">
                     <h3>Contact Us</h3>
                     <br>
                     <p>Somthing about us. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
```
Comment (chayan007) [inline templates/pages/public/home.html:117]: Put actual content in this block. Refer to Ankit !

Comment (prihit) [reply templates/pages/public/home.html:117]: okay

File: templates/layouts/public.html (hunk 2 above)
Comment (prihit) [reply templates/layouts/public.html]: done

File: base/views.py [hunk 4]
```diff
@@ -18,7 +18,7 @@ def custom_internal_error(request, *args, **argv):
 def home(request):
     colleges = College.objects.filter(
         is_top=True,
-    )[:9]
+    )
```
Comment (chayan007) [inline base/views.py]: No need to load all colleges with `is_top=True`. Load first 9 or 12 based on your need

File: templates/pages/public/home.html (hunk 3 above)
Comment (prihit) [reply templates/pages/public/home.html]: Done

File: templates/pages/public/home.html [hunk 5]
```diff
@@ -1212,20 +140,29 @@ <h4>Pages default text-field design</h4>
       <div class="card-body">
         <p>Fancy social icons? here are some of pre-made icons and buttons that are done which can come in handy</p>
```
Comment (chayan007) [inline templates/pages/public/home.html:140]: Use proper content. Take it from Ankit !

Comment (chayan007) [inline templates/pages/public/home.html:140]: put the entire social icons div inside the footer block. Pointer `templates/layouts/public.html L#59`

Comment (chayan007) [reply templates/pages/public/home.html:140]: Do not erase the existing footer though, keep it at the bottom.

Comment (chayan007) [reply templates/pages/public/home.html:140]: Use these buttons for social media follow links and not for registration. Actually these should be icons, not even buttons.

Comment (chayan007) [reply templates/pages/public/home.html:140]: Social buttons should be in the auth pages (totally a different pipeline object)

**Review by chayan007 — CHANGES_REQUESTED** [review]

File: templates/pages/public/home.html (hunk 5 above)
Comment (prihit) [reply templates/pages/public/home.html:140]: ok

Comment (prihit) [reply templates/pages/public/home.html:140]: cool

**Pull request merged**